modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/CheckpointProgressImpl.java

```diff
@@ -273,35 +276,59 @@ void pagesToWrite(@Nullable CheckpointDirtyPages pageToWrite) {
     }
 
     /**
-     * Callback at the beginning of checkpoint processing of a partition, for example, when writing dirty pages or executing a fsync.
+     * Blocks physical destruction of partition.
+     *
+     * <p>When the intention to destroy partition appears, {@link FilePageStore#isMarkedToDestroy()} {@code == true} and invoked
+     * {@link PersistentPageMemory#invalidate(int, int)} at the beginning. And if there is a block, it waits for unblocking. Then it
```

Comment:
"method invoked", not "invoked method", please use a proper order of words

Reply:
fixed